src/main/java/shop/kkeujeok/kkeujeokbackend/member/follow/api/dto/request/FollowReqDto.java

```diff
@@ -0,0 +1,16 @@
+package shop.kkeujeok.kkeujeokbackend.member.follow.api.dto.request;
+
+import shop.kkeujeok.kkeujeokbackend.member.domain.Member;
+import shop.kkeujeok.kkeujeokbackend.member.follow.domain.Follow;
+import shop.kkeujeok.kkeujeokbackend.member.follow.domain.FollowStatus;
+
+public record FollowReqDto(
+        Long memberId) {
```

Comment:
코드 컨벤션 맞춰주세요!

```suggestion
        Long memberId
) {
```

다른 reqDto에도 있더라고요!